core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1104,7 +1104,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
       if (!fs.exists(hadoopPath)) {
         throw new FileNotFoundException(s"Added file $hadoopPath does not exist.")
       }
-      val isDir = fs.isDirectory(hadoopPath)
```

Comment:
In case you're wondering: no this wasn't one of those things deprecated in Hadoop 2.x; this was deprecated in 1.0.4 even!